test/libsolidity/SemanticTest.cpp

```diff
@@ -119,7 +144,13 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 	return result;
 }
 
-TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _linePrefix, bool _formatted, bool _compileViaYul, bool _compileToEwasm)
+void SemanticTest::addBuiltin(std::string _module, std::string _function, std::shared_ptr<Builtin> _builtin)
```

Comment:
```suggestion
void SemanticTest::addBuiltin(string _module, string _function, shared_ptr<Builtin> _builtin)
```

Reply:
Could not do it for `std::shared_ptr`, because it is getting ambiguous because of `boost::shared_ptr`.